### PR TITLE
Add OpenStack tutorial

### PR DESCRIPTION
**Pull request by 14rcole**

### CODE REVIEW

File: docs/source/tutorials/openstack.rst [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+
+Creating a Layout
+-----------------
+
+LinchPin can use layouts to describe what an Ansible inventory might look like after provisioning.  Layouts can include information such as IP addresses, zones, and FQDNs.  Under the simple key, put the following data:
+
+.. code:: yaml
+
+	layout:
```
Comment (p3ck) [inline docs/source/tutorials/openstack.rst]: This should be indented the same as topology above right?  it looks like it should be pushed over a little more?

File: docs/source/tutorials/openstack.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+			host_groups:
+			  - frontent
+	  host_groups:
+		all:
+		  vars:
+			ansible_user: root
+		frontend:
+		  vars:
+
+After provisioning the hosts, LinchPin will through each host type in the inventory_layout, pop :code:`count` hosts off of the list, and add them to the relevant host groups.  The :code:`host_groups` section of the layout is used to set environment variables for each of the hosts in a given host group
```
Comment (p3ck) [inline docs/source/tutorials/openstack.rst]: will through -> will iterate through?

File: docs/source/tutorials/openstack.rst (hunk 1 above)
Comment (14rcole) [reply docs/source/tutorials/openstack.rst]: Yes, I'll fix that

**Comment by 14rcole** [comment]
@p3ck @samvarankashyap is this good now

**Comment by 14rcole** [comment]
[merge]

**Pull request merged**